app/Jobs/ImportEmailSubscription.php

```diff
@@ -108,9 +108,11 @@ private function createUser()
     private function updateUser($user)
     {
         $existingTopics = ! empty($user->email_subscription_topics) ? $user->email_subscription_topics : [];
+        $newTopics = array_unique(array_merge($existingTopics, $this->email_subscription_topics));
```

Comment:
@katiecrane I was able to add duplicate subscription topics, this seems like it'd be a useful sanity check to add into Northstar validation?

Reply:
@aaronschachter that is a great call! right now Northstar assumes it is getting the complete list of subscription topics for a user with no duplicates.

Reply:
also wow `array_unique`, didn't realize that existed, now there's some other code I can simplify 😈

Reply:
I made a barebones card for this Northstar issue so we don't forget about it! https://www.pivotaltracker.com/story/show/165074676